Providers/Profile.swift

```diff
@@ -275,6 +275,10 @@ public class BrowserProfile: Profile {
             }
             // Set the default homepage.
             prefs.setString(PrefsDefaults.ChineseHomePageURL, forKey: PrefsKeys.KeyDefaultHomePageURL)
+
+            if prefs.stringForKey("NewTabPrefKey") == nil {
```

Comment:
I covered this in the commit message. TL;DR: I don't like how why do setting of pref defaults in the profile.